src/quick-start.md

```diff
@@ -128,11 +129,15 @@ Overview of the attributes of the `ClusterAdmissionPolicy` resource:
 | | | - `Ignore`: an error calling the webhook is ignored and the API request is allowed to continue |
 | | | - `Fail`: an error calling the webhook causes the admission to fail and the API request to be rejected |
 
-The complete documentation of this Custom Resource can be found [here](https://github.com/kubewarden/kubewarden-controller/blob/main/docs/crds/README.asciidoc) or on [docs.crds.dev](https://doc.crds.dev/github.com/kubewarden/kubewarden-controller).
-
 > **NOTE:** The  `ClusterAdmissionPolicy` resources are registered with a `*` webhook `scope`, which means that registered webhooks will forward all requests matching the given `resources` and `operations` -- either namespaced (in any namespace), or cluster-wide resources.
 
-> **NOTE:** The `ClusterAdmissionPolicy` resource is cluster-wide. There are plans to also provide a namespaced version that will only impact registered namespaced resources on its own namespace.
+### AdmissionPolicy
+
+`AdmissionPolicy` is a namespaced resource. Its resources are registered with a `Namespaced` webhook `scope`, which means that registered webhooks will forward all requests matching the given `resources` and `operations` to the `AdmissionPolicy` namespace. Otherwise there is no difference between `AdmissionPolicy` and `ClusterAdmissionPolicy`.
```

Comment:
I think we might need to reframe this entirely as below 
```suggestion
`AdmissionPolicy` is a namespace-wide resource. This means that its resources are registered with a `namespace` webhook `scope` that will forward all requests matching the given `resources` and `operations` to the `AdmissionPolicy` namespace. Other than that, there are no functional differences between the `AdmissionPolicy` and `ClusterAdmissionPolicy` resources.
```

Reply:
I would leave the scope as 'Namespaced' since a scope can be 'Cluster' or 'Namespaced' [more info](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#configure-admission-webhooks-on-the-fly)
Other suggestion done in https://github.com/kubewarden/docs/commit/3884c372cb0809d18ac9e102d03aef8387e41992